src/System.IO.FileSystem.Primitives/src/System.IO.FileSystem.Primitives.csproj

```diff
@@ -4,24 +4,31 @@
   <PropertyGroup>
     <ProjectGuid>{6C05678E-394C-4CFF-B453-A18E28C8F2C3}</ProjectGuid>
     <AssemblyName>System.IO.FileSystem.Primitives</AssemblyName>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' != 'netcore50aot'">true</IsPartialFacadeAssembly>
```

Comment:
I do not think we should be bothering with keeping the netcore50 build live. Delete it instead - it is what we are doing everywhere else.